### DISCUSSION TITLE
Download Multiplay License ID return empty

### DISCUSSION THREAD

**Discussion by ToffelsKater** [pixiv/vroid-sdk-developers#169] [Q&A] [answered]
Hey quick question this node returns empty every time i try calling it, it works when i use the example level "AuthLevel"
So my question is which requirements are need to be fulfilled to get it to work?
![image](https://github.com/user-attachments/assets/8bda15b4-bd2c-49ee-8110-c8e9f767248f)

**Comment by mameoPix** [comment, marked as answer]
Sorry for the delay in replying.
Probably the following Log is output at runtime.
`LogVRoid: Issue of model license Id for multiplayer is skipped because the multiplayer flag of VRoidGameInstanceSubSystem is not valid.
`

There are several conditions that must be met for DownloadMultiplayLicenseId to function properly.
1. The IsMultiplay flag in the UVRoidGameInstanceSubsystem must be valid.
　The IsMultiplay flag is enabled when initializing WBP_Session used in Session level.
2. You must enable multiplay in Scope from the VRoidHub application settings.
　To do so, you must apply to use Multiplay in advance.

I am sorry to say that details can be found in the following document in the section titled “「Multiplay機能を使用するには」”. 
The English documentation is not yet ready and we apologize for the inconvenience.
https://vroid.notion.site/VRoid-SDK-for-Unreal-Engine-21321b5e31b247e99600f270b3414d4e#dbd9ec3220d445e3b0e31123a3ea8192

When these conditions are met, the License information to synchronize with other Clients during Multiplay will be stored in the VRoidPlayerControllerBase.

**Reply by ToffelsKater** [reply]
Thank you very much for your reply, My Application is already approved for Multiplay, I will try the other steps mentioned and close the ticket if it works

**Reply by ToffelsKater** [reply]
Thank you, it seems like I was missing this flag
![image](https://github.com/user-attachments/assets/727f8f96-56c9-4964-a74f-d84d1a76f964)
